### PR TITLE
add miscellaneous-topics.md

### LINKED ISSUE #133: Add 'Miscellaneous topics':

### PR DESCRIPTION
**Pull request by zhangyangyu**
### What issue does this PR solve?

- close #133

### What is changed:

This PR adds a template. If there is more info just add to it.

### CODE REVIEW

File: src/contribute-to-tidb/miscellaneous-topics.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Miscellaneous Topics
+
+## Communication channels
+
+- [TiDB Internals Forum](https://internals.tidb.io): TiDB hosts a Discourse instance for TiDB development discussions. It has many separate categories for different topics. You can discuss anything about TiDB development and community in the forum.
+
+## Related projects
+
+- [TiKV](https://github.com/tikv/tikv): TiKV is an open-source, distributed, and transactional key-value database. It is used by TiDB as the storage layer.
+- [Talent Plan](https://university.pingcap.com/talent-plan/): Talent Plan is an open source training program initiated by PingCAP. It aims to create or combine some open source learning materials for people interested in open source, distributed systems, Rust, Golang, and other infrastructure knowledges.
```
Comment (sunxiaoguang) [inline src/contribute-to-tidb/miscellaneous-topics.md:10]:
```suggestion
- [Talent Plan](https://university.pingcap.com/talent-plan/): Talent Plan is an open source training program initiated by PingCAP. It aims to create and combine some open source learning materials for people interested in open source, distributed systems, Rust, Golang, and other infrastructure knowledges.
```

**Review by sunxiaoguang — COMMENTED** [review]
Correct me if I didn't understand it correctly.

File: src/contribute-to-tidb/miscellaneous-topics.md (hunk 1 above)
Comment (zhangyangyu) [reply src/contribute-to-tidb/miscellaneous-topics.md:10]: I +1 for `and` over `or`. I copied this introduction from https://github.com/pingcap/talent-plan .

**Review by tisonkun — APPROVED** [review]
LGTM. Merging...

**Pull request merged**